src/test/java/org/fogbowcloud/manager/core/plugins/behavior/federationidentity/ldap/LdapLocalIdentityPluginTest.java

```diff
@@ -23,121 +25,83 @@
+    Map<String, String> userCredentials = new HashMap<String, String>();
+    
     @Before
     public void setUp() throws Exception {
-
         HomeDir.getInstance().setPath("src/test/resources/private");
         PropertiesHolder propertiesHolder = PropertiesHolder.getInstance();
         Properties properties = propertiesHolder.getProperties();
-        properties.put(IDENTITY_URL_KEY, KEYSTONE_URL);
-
+        properties.put(IDENTITY_URL_KEY, this.KEYSTONE_URL);
         this.ldapStoneIdentity = Mockito.spy(new LdapIdentityPlugin());
-        Mockito.doReturn(MOCK_SIGNATURE)
-                .when(ldapStoneIdentity)
+        Mockito.doReturn(this.MOCK_SIGNATURE)
+                .when(this.ldapStoneIdentity)
                 .createSignature(Mockito.any(JSONObject.class));
-    }
 
+        this.userCredentials.put(LdapIdentityPlugin.CRED_USERNAME, this.name);
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PASSWORD, this.password);
+        this.userCredentials.put(LdapIdentityPlugin.CRED_AUTH_URL, "ldapUrl");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_LDAP_BASE, "ldapBase");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_LDAP_ENCRYPT, "");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PRIVATE_KEY, "private_key_path");
+        this.userCredentials.put(LdapIdentityPlugin.CRED_PUBLIC_KEY, "public_key_path");
+    }
+    
+    //test case: check if the access id information is correct when the user credentials are correct.
     @Test
     public void testCreateToken() throws Exception {
-
-        String name = "ldapUser";
-        String password = "ldapUserPass";
-        String userName = "User Full Name";
-
-        Map<String, String> userCredentials = new HashMap<String, String>();
-        userCredentials.put(LdapIdentityPlugin.CRED_USERNAME, name);
-        userCredentials.put(LdapIdentityPlugin.CRED_PASSWORD, password);
-        userCredentials.put(LdapIdentityPlugin.CRED_AUTH_URL, "ldapUrl");
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_BASE, "ldapBase");
-        userCredentials.put(LdapIdentityPlugin.CRED_LDAP_ENCRYPT, "");
-        userCredentials.put(LdapIdentityPlugin.CRED_PRIVATE_KEY, "private_key_path");
-        userCredentials.put(LdapIdentityPlugin.CRED_PUBLIC_KEY, "public_key_path");
-
-        Mockito.doReturn(userName)
-                .when(ldapStoneIdentity)
-                .ldapAuthenticate(Mockito.eq(name), Mockito.eq(password));
-
-        String federationTokenValue = ldapStoneIdentity.createFederationTokenValue(userCredentials);
+    	//set up
+        Mockito.doReturn(this.userName)
+                .when(this.ldapStoneIdentity)
+                .ldapAuthenticate(Mockito.eq(this.name), Mockito.eq(this.password));
+        
+        //exercise
+        String federationTokenValue = this.ldapStoneIdentity.createFederationTokenValue(userCredentials);
+        
+        //verify
         String decodedAccessId = decodeAccessId(federationTokenValue);
```

Comment:
`decodeAccessId` method could be private.

Reply:
Thanks for catching that.